bubblewrap.c

```diff
@@ -219,6 +220,7 @@ usage (int ecode, FILE *out)
            "    --info-fd FD                 Write information about the running container to FD\n"
            "    --new-session                Create a new terminal session\n"
            "    --die-with-parent            Kills with SIGKILL child process (COMMAND) when bwrap or bwrap's parent dies.\n"
+           "    --no-reaper                  Do not install a reaper process with PID=1\n"
```

Comment:
I dunno...it feels like we're optimizing for the wrong thing here.  We can easily error out if the user doesn't pass `--unshare-pid` too.

But if I was looking at the `--help` output I would be a lot less likely to find `--unshare-pid-noinit` if I was looking for this functionality.

`--as-pid1  The executed program will be PID 1 (suppress builtin init)`
vs
`--unshare-pid-noinit Do not install a reaper process with PID=1`

?